source/includes/harvest/_users.md

```diff
@@ -86,6 +86,63 @@ List all of an organization's Greenhouse users.
 <br>
 [See noteworthy response attributes.] (#the-user-object)
 
+## GET: List Future Job Permissions for User
+
+```shell
+curl 'https://harvest.greenhouse.io/v1/users/{id}/permissions/future_jobs'
+-H "Authorization: Basic MGQwMzFkODIyN2VhZmE2MWRjMzc1YTZjMmUwNjdlMjQ6"
+```
+
+```json
+[
+  {
+    "id": 87819,
+    "office_id": null,
+    "department_id": null,
+    "user_role_id": 4730
+  },
+  {
+    "id": 92834,
+    "office_id": 8232,
+    "department_id": null,
+    "user_role_id": 4730
+  },
+  {
+    "id": 82129,
+    "office_id": 8232,
+    "department_id": 92921,
+    "user_role_id": 4730
+  }
+]
+```
+
+List the criteria for job permissions that will be granted to the user when jobs created in the future match that criteria.
```

Comment:
Ok. I was hesitant to say that it was listing permissions because it's technically not listing permissions but it resulted in a confusing statement. And introducing a new word like criteria is probably a bad idea.